### ISSUE TITLE
How to use from typescript?

### ISSUE THREAD

**Issue by hhanh00** [deepstreamIO/deepstream.io-client-js#503]
I tried importing as

```
import * as deepstream from '@deepstream/client'
const client = deepstream('localhost:6020')
```

or as

```
import * as deepstream from '@deepstream/client'
const client = new deepstream.Client('localhost:6020')
```

and as

```
import deepstream = require('@deepstream/client')
const client = new deepstream.Client('localhost:6020')
```

I either get a compilation error

`This expression is not callable`
or a runtime error `deepstream.Client is not a constructor`

Only way that works is the old fashioned js 
```
const deepstream = require('@deepstream/client')
const client = deepstream('localhost:6020')
```

Is it the recommended way?

**Comment by yasserf** [comment]
hey! Yeah sorry this is an actual issue.

What I do for now is:

```
const deepstream = require('@deepstream/client')
import { Client as DeepstreamClient } from '@deepstream/client'
```

Need to figure out how to fix it in the SDK. Thanks for raising!

**Comment by yasserf** [comment]
This is fixed in V5 (to be released), since it  is a minor breaking change. Thanks for raising!

**Comment by hhanh00** [comment]
Thank you for getting back to me and for this great library!

**Comment by yasserf** [comment]
👍 

Fixed in 5.0
